sdk/appconfiguration/azure-appconfiguration-provider/tests/async_preparers.py

```diff
@@ -12,6 +12,9 @@ async def wrapper(*args, **kwargs):
         appconfiguration_connection_string = kwargs.pop("appconfiguration_connection_string")
         kwargs["appconfiguration_connection_string"] = appconfiguration_connection_string
 
+        appconfiguration_keyvault_secret_url = kwargs.pop("appconfiguration_keyvault_secret_url")
+        kwargs["appconfiguration_keyvault_secret_url"] = appconfiguration_keyvault_secret_url
```

Comment:
What's this for?

Reply:
I actually don't know what that section does. I was just copying what was in the other preparers.